samples: MITM numeric comparison pairing mode (#948)
NordicPlayground/fw-nrfconnect-nrf: samples/bluetooth/peripheral_uart/README.rst

LED 2:
   * On when connected.

> BUTTON 1:

Reviewer: ```suggestion
Button 1:
```

>    * Confirm the passkey value which is printed on the COM listener to pair/bond with other device.

Reviewer: ```suggestion
   * Confirm the passkey value that is printed on the COM listener to pair/bond with the other device.
```

> BUTTON 2:

Reviewer: ```suggestion
Button 2:
```

>    * Reject the passkey value which is printed on the COM listener to pair/bond with other device.

Reviewer: ```suggestion
   * Reject the passkey value that is printed on the COM listener to prevent pairing/bonding with the other device.
```

#. Observe that the text "Starting Nordic UART service example" is printed on the COM listener running on the computer.
#. Connect to the device using nRF Connect for Mobile.
   Observe that LED 2 is on.

> #. Optionally pair/bond with the device with MITM protection, to confirm press Button 1 on the device and accept the passkey value on a phone.

Reviewer: ```suggestion
#. Optionally, pair/bond with the device with MITM protection.
   To confirm pairing/bonding, press Button 1 on the device and accept the passkey value on the smartphone.
```

>    If tested with :ref:`central_uart` use the Button 1 on the both devices to confirm bonding.

Reviewer: ```suggestion
   If tested with :ref:`central_uart`, use Button 1 on both devices to confirm bonding.
```
